### PR TITLE
chore: Update WAF definition schema

### PR DESCRIPTION
**Pull request by ronaldofsilveira**

### CODE REVIEW

**Review by gomes-fdr — APPROVED** [review]

File: waf.yaml [hunk 1]
```diff
@@ -111,8 +151,299 @@ paths:
...
+    get:
+      tags:
+      - WAF
+      summary: list all Rule Sets associated to a Web Application Firewall (WAF) in an account.
+      operationId: listAllWAFRulesets
+      parameters:
+      - in: query
+        name: order_by
+        description: Identifies which property the return should be sorted by.
+        required: true
```
Comment (diegoazion) [inline waf.yaml:164]: Another question are these parameters mandatory by the API?

File: waf.yaml [hunk 2]
```diff
@@ -111,8 +151,299 @@ paths:
...
+        schema:
+            type: string
+            enum: [id,name,mode,active,sql_injection,sql_injection_sensitivity,remote_file_inclusion,remote_file_inclusion_sensitivity,directory_traversal,directory_traversal_sensitivity,cross_site_scripting,cross_site_scripting_sensitivity,evading_tricks,evading_tricks_sensitivity,file_upload,file_upload_sensitivity,unwanted_access,unwanted_access_sensitivity,identified_attack,identified_attack_sensitivity]
+            default: "name"
+      - in: query
+        name: sort
+        description: Defines whether objects are shown in ascending or descending order depending on the value set in order_by.
+        required: true
```
Comment (diegoazion) [inline waf.yaml:172]: Same from another are these parameters mandatory by the API?

File: waf.yaml [hunk 3]
```diff
@@ -111,8 +151,299 @@ paths:
...
+        schema:
+            type: string
+            enum: ["asc", "desc"]
+            default: "asc"
+      - in: query
+        name: page
+        description: Identifies which page should be returned, if the return is paginated.
+        required: true
```
Comment (diegoazion) [inline waf.yaml:180]: Same from another are these parameters mandatory by the API?

File: waf.yaml [hunk 4]
```diff
@@ -111,8 +151,299 @@ paths:
...
+        schema:
+            type: integer
+            default: 1
+      - in: query
+        name: page_size
+        description: Identifies how many items should be returned per page.
+        required: true
```
Comment (diegoazion) [inline waf.yaml:187]: Same from another are these parameters mandatory by the API?

**Review by azionhsantos — APPROVED** [review]

**Review by AndreSilveiraAzion — APPROVED** [review]

**Pull request merged**